### PR TITLE
Update documentation regarding no longer processing duplicate webhooks.

### PR DESCRIPTION
**Pull request by hashimoon**
### What

Added documentation explaining that we do not process webhooks for duplicate commits

Pages changed:
- /terraform/cloud-docs/vcs
- /terraform/cloud-docs/run/ui
- /terraform/cloud-docs/vcs/troubleshooting

Related Atlas PR: https://github.com/hashicorp/atlas/pull/14529

### Why

This reflects the change in default behavior for no longer processing webhooks for duplicate commit SHA's. Previously, we would process any webhook received even if we processed it in the past, which resulted in customer data loss as a result of an old webhook being processed that removed infrastructure.

### Screenshots
Screen Shot 2023-02-08 at 2 47 31 PM
Screen Shot 2023-02-08 at 2 45 34 PM
Screen Shot 2023-02-08 at 2 44 21 PM

----------

### Merge Checklist
_If items do not apply to your changes, add (N/A) and mark them as complete._

#### Pull Request
- [x] One or more labels describe the type of change (e.g. clarification) and associated product (e.g. tfc).
- [x] Description links to related pull requests or issues, if any.

#### Content
- [x] Redirects have been added for moved, renamed, or deleted pages. This requires a separate PR in the [`terraform-website` repository](https://github.com/hashicorp/terraform-website) `redirects.next.js` file.
- [x] API documentation and the API Changelog have been updated. 
- [x] Links to related content where appropriate (e.g., API endpoints, permissions, etc.).
- [x] Pages with related content are updated and link to this content when appropriate.
- [x] Sidebar navigation files have been updated for added, deleted, reordered, or renamed pages.
- [x] New pages have metadata (page name and description) at the top.
- [x] New images are 2048 px wide. They have HashiCorp standard annotation color (#F92672) and format (rectangle with rounded corners), blurred sensitive details (e.g. credentials, usernames, user icons), and descriptive alt text in the markdown for accessibility.
- [x] New code blocks have the correct syntax and line breaks to eliminate horizontal scroll bars.
- [x] UI elements (button names, page names, etc.) are bolded.
- [x] The Vercel website preview successfully deployed.

#### Reviews
- [ ] I or someone else reviewed the content for technical accuracy.
- [x] I or someone else reviewed the content for typos, punctuation, spelling, and grammar.

### CODE REVIEW

File: website/docs/cloud-docs/vcs/troubleshooting.mdx [hunk 1]
```diff
@@ -87,6 +87,8 @@ The fix is to update the OAuth Callback URL in your VCS provider to use app.terr
 
 A workspace with no runs will not accept new runs from a VCS webhook. You must queue at least one run manually.
 
+A workspace will not process a webhook if the workspace previously processed a webhook with the same commit SHA and created a run. To trigger a run, create a new commit.
```
Comment (paulanunda) [inline website/docs/cloud-docs/vcs/troubleshooting.mdx:90]: We should mention that the user should look at the VCS Event page to determine if a duplicate commit was received and not processed.

**Comment by paulanunda** [comment]
We should update the VCS Event viewer documentation [here](https://developer.hashicorp.com/terraform/cloud-docs/vcs#viewing-events)

**Comment by paulanunda** [comment]
I think we should perhaps add an additional note about ignoring duplicate webhooks [here](https://developer.hashicorp.com/terraform/cloud-docs/vcs#webhooks)

**Review by paulanunda — APPROVED** [review]

File: website/docs/cloud-docs/vcs/index.mdx [hunk 2]
```diff
@@ -108,9 +108,7 @@ For complete details, click the link for your VCS provider:
 
 ## Viewing events
 
--> **Note**: The VCS Event Viewer is still in beta as support is being added for additional VCS providers. Currently only GitLab.com connections established after December 2020 are supported.
-
-VCS events describe changes within your organization for VCS-related actions, such as creating a new VCS provider (also called an OAuth Client), loading repositories while creating a new workspace, or processing incoming webhooks. Displayed events are limited to the past 10 days.
+VCS events describe changes within your organization for VCS-related actions, such as when a webhook is received with a previously processed commit SHA. Displayed events are limited to the past 30 days.
```
Comment (judithpatudith) [inline website/docs/cloud-docs/vcs/index.mdx:111]:
```suggestion
VCS events describe changes within your organization for VCS-related actions, such as when Terraform Cloud receives a webhook with a previously processed commit SHA. The VCS events page only displays events from the past 30 days.
```

File: website/docs/cloud-docs/run/ui.mdx [hunk 3]
```diff
@@ -31,6 +31,8 @@ A workspace is linked to one branch of a VCS repository and ignores changes to o
 
 -> **Note:** A workspace with no runs will not accept new runs via VCS webhook. At least one run must be manually queued to confirm that the workspace is ready for further runs.
 
+-> **Note:** A workspace will not process a webhook if the workspace previously processed a webhook with the same commit SHA and created a run. To trigger a run, create a new commit. If a workspace receives a webhook with a previously processed commit, a new event will be added to the [VCS Events](/cloud-docs/vcs#viewing-events) page documenting the received webhook.
```
Comment (judithpatudith) [inline website/docs/cloud-docs/run/ui.mdx:34]:
```suggestion
A workspace will not process a webhook if the workspace previously processed a webhook with the same commit SHA and created a run. To trigger a run, create a new commit. If a workspace receives a webhook with a previously processed commit, Terraform Cloud will add a new event to the [VCS Events](/cloud-docs/vcs#viewing-events) page documenting the received webhook.
```

File: website/docs/cloud-docs/vcs/troubleshooting.mdx [hunk 4]
```diff
@@ -87,6 +87,8 @@ The fix is to update the OAuth Callback URL in your VCS provider to use app.terr
 
 A workspace with no runs will not accept new runs from a VCS webhook. You must queue at least one run manually.
 
+A workspace will not process a webhook if the workspace previously processed a webhook with the same commit SHA and created a run. To trigger a run, create a new commit. If a workspace receives a webhook with a previously processed commit, a new event will be added to the [VCS Events](/cloud-docs/vcs#viewing-events) page documenting the received webhook.
```
Comment (judithpatudith) [inline website/docs/cloud-docs/vcs/troubleshooting.mdx:90]:
```suggestion
A workspace will not process a webhook if the workspace previously processed a webhook with the same commit SHA and created a run. To trigger a run, create a new commit. If a workspace receives a webhook with a previously processed commit, a Terraform Cloud will add a new event to the [VCS Events](/cloud-docs/vcs#viewing-events) page documenting the received webhook.
```

**Review by judithpatudith — APPROVED** [review]
Looks good, thank you!

**Pull request merged**